test/e2e/live_iso_test.go

```diff
@@ -0,0 +1,69 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	bmov1alpha1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("When testing live iso [live-iso]", func() {
+	liveIsoTest()
+})
+
+func liveIsoTest() {
+	var (
+		liveIsoImageURL = e2eConfig.GetVariable("LIVE_ISO_IMAGE")
```

Comment:
I would stick to capitalizing the acronym (ISO) in the code as it is a best practice in Go. 
```suggestion
		liveISOImageURL = e2eConfig.GetVariable("LIVE_ISO_IMAGE")
```